dog-lover/src/utils/index.tsx

```diff
@@ -0,0 +1,13 @@
+export const randomItem = <T,>(items: T[]): T => {
+  return items[Math.floor(Math.random() * items.length)];
+};
+
+export const nextItem = <T,>(items: T[], current: T): T => {
```

Comment:
Can you explain why you used generics here? It doesn't seem to be used somewhere

Reply:
I like to keep any functions in the utils folder as reusable throughout the application as possible. In that way and as the project grows larger, if I need to reuse them with different types of input, I can easily grab and use them right away!

Thanks again for your time!